docs/getting_started.md

```diff
@@ -0,0 +1,114 @@
+# Getting Started With istio-csr
+
+This guide will run through installing and using istio-csr from scratch. We'll use [kind](https://kind.sigs.k8s.io/) to create a new cluster locally in Docker, but this guide should work on any cluster.
```

Comment:
There are extra steps that must be done so that the cluster can be ready before insalling Istio. The Istio docs call these setup tasks "Plaform Setup" (add policies on openshift, add firewall rules and such). Should we mention these "Platform Setup" steps here?

```suggestion
This guide will run through installing and using istio-csr from scratch. We'll use [kind](https://kind.sigs.k8s.io/) to create a new cluster locally in Docker, but this guide should work on any cluster as long as the [Platform Setup](https://istio.io/latest/docs/setup/platform-setup/) has been performed. Note that if you are following the Platform Setup for OpenShift, do not run the `istioctl install` command listed in the Istio documentation; we will be performing later in this getting started guide.
```

Reply:
Good suggestion, thanks :+1: